JSDemos/Demos/Charts/AxisLabelsTemplates/description.md

```diff
@@ -0,0 +1,5 @@
+Axes [labels](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) are visual elements that display values of the [major axis ticks](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/tick/).
```

Comment:
> Axes [labels](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/)

- It should be "ax**i**s".
- We give links to labels further. Let's remove this link.